docs/workflows/phylogenetic_construction/augur.md

```diff
@@ -290,6 +290,7 @@ The Nextstrain team hosts documentation surrounding the Augur workflow → Auspi
 | auspice_input_json | File | JSON file used as input to Auspice |
 | combined_assemblies | File | Concatenated FASTA file containing all samples |
 | distance_tree | File | The distance tree created in Newick (.nwk) format |
+| iqtree_model_used | String | The iqtree model used during augur tree |
```

Comment:
just update this doc to be augur_iqtree_model_used and then i'll merge 👍

Reply:
Good catch